File: 2574-left-and-right-sum-differences/2574-left-and-right-sum-differences.java
class Solution {
    public int[] leftRightDifference(int[] nums) {

          
        int[] arr1 = new int[nums.length];
        int[] arr2 = new int[nums.length];
        
        for(int i=1; i<nums.length; i++)
        {
         
           arr1[i] += nums[i-1]+arr1[i-1];
          
           
        }
        for(int j= nums.length-2; j>=0; j--)
        {
           
           arr2[j] += nums[j+1]+arr2[j+1];
        
           
        }
        int [] res = new int[nums.length];
        
        for(int k =0; k<nums.length; k++)
        {
                res[k] = Math.abs(arr2[k]-arr1[k]);
              
        }
        
     
        return res;
    }
}

File: 2574-left-and-right-sum-differences/2574-left-and-right-sum-differences.py
class Solution:
    def leftRightDifference(self, nums: List[int]) -> List[int]:

        l1 = []
        
        for i in range(len(nums)):
            l =0
            r = 0
            for j in range(0, i):
                l += nums[j]
            for k in range(i+1, len(nums)):
                r += nums[k]
            l1.append(abs(l-r))
        return l1

        